2007-11-18  Aurelien Jarno  <aurelien@aurel32.net>

	* malloc/malloc.h: Define __need_size_t and __need_ptrdiff_t.

Index: malloc/malloc.h
===================================================================
RCS file: /cvs/glibc/libc/malloc/malloc.h,v
retrieving revision 1.30
diff -u -d -p -r1.30 malloc.h
--- malloc/malloc.h	19 Jul 2007 17:05:07 -0000	1.30
+++ malloc/malloc.h	28 Nov 2007 22:57:02 -0000
@@ -22,6 +22,8 @@
 #define _MALLOC_H 1
 
 #include <features.h>
+#define __need_size_t
+#define __need_ptrdiff_t
 #include <stddef.h>
 # define __malloc_ptr_t  void *
 
